news/2024/2024-08-12-the-followpoint-motion-the-ultimate-catcher.md

```diff
@@ -0,0 +1,294 @@
+---
+layout: post
+title: "The Followpoint: Motion, the Ultimate Catcher"
+date: 2024-08-12 17:00:00 +0000
+series: The Followpoint
+---
+
+The Followpoint is back with another interview! Learn more about Motion and his opinions as a true master of osu!catch game mode.
+
+![](/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/banner.jpg)
+
+Welcome or welcome back to The Followpoint, a series where we interview various community figures and players! This time we interviewed [Motion](https://osu.ppy.sh/users/3885626), a player who has mastered osu!catch to the point where he managed to take #1 rank from [YesMyDarknesss](https://osu.ppy.sh/users/4158549), who had it for over 3 years!
+
+So, let's have a read!
+
+---
+
+<style>
+    .news-chat-quote__avatar {
+        float: left;
+        width: 40px;
+        height: 40px;
+        border-radius: 50%;
+        margin-left: -50px;
+    }
+
+    .news-chat-quote__text-container {
+        margin-left: 50px;
+    }
+
+    .news-chat-quote__username {
+        font-weight: 600;
+        margin-bottom: 2px;
+    }
+
+    .news-chat-quote__colour-lvd {
+        color: #FFD1DC;
+    }
+
+    .news-chat-quote__colour-no-group {
+        color: #FFFFFF;
+    }
+</style>
+
+<div class="news-chat-quote__text-container">
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Hi Motion, let's start with an introduction, how would you introduce yourself to people who don't know you?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3885626" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-Motion.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3885626">Motion</a></p>
+
+Hello everyone, my name is Motion, I've played ctb for a decade and hit #1 recently.
```

Comment:
```suggestion
Hello everyone, my name is Motion, I've played osu!catch for a decade and hit #1 recently.
```

Reply:
Personally don't see an issue here, especially because it's their answer